database/dbconnection.js

```diff
@@ -2,8 +2,10 @@ const mongoose = require('mongoose');
 
 require('dotenv').config();
 
-if (!process.env.DATABASE_URL) {
+const { DATABASE_URL } = process.env;
```

Comment:
Nice destructuring :wink:

Reply:
I really needed to feel like I had added to the code somehow!